services/utils/formatDate.js

```diff
@@ -0,0 +1,14 @@
+export function formatDate(date, locale = "en-US") {
```

Comment:
It looks like we were using `en-GB` why switch to `en-US`?

Reply:
It was a mistake which I forgot to change.

Reply:
Will do it right away